lib/compiler-singlepass/Cargo.toml

```diff
@@ -26,6 +26,7 @@ hashbrown = { version = "0.11", optional = true }
 lazy_static = "1.4"
 memoffset = "0.6"
 more-asserts = "0.2"
+num_enum = "0.5"
```

Comment:
we use `strum` in nearcore, so lets stick to that to avoid inflating Cargo.lock (although at a glance num_enum looks more tame)